CreeDictionary/tests/conftest.py

```diff
@@ -56,33 +54,3 @@ def random_lemmas(draw) -> Wordform:
     """
     lemmas = Wordform.objects.filter(is_lemma=True, as_is=False)
     return draw(sampled_from(list(lemmas)))
-
-
-@pytest.fixture(scope="session")
-def one_hundredth_xml_dir(topmost_datadir) -> Path:
```

Comment:
We have a test database now, not using this anymore

Reply:
I approve and appreciate this change, but I don't think it belongs in _this_ PR. Please strive to keep PRs super focused on one thing.